src/angular-app/bellows/shared/sound-player.component.ts

```diff
@@ -51,6 +54,10 @@ export class SoundController implements angular.IController {
       this.isUserMovingSlider = true;
     });
 
+    // this.audioElement.addEventListener('ended', () => {
```

Comment:
can these be removed?

Reply:
Yes, thanks for seeing these and commenting.